src/local-editor-service.ts

```diff
@@ -140,6 +139,24 @@ export class LocalEditorService extends EditorService {
     };
   }
 
+  _generateAutoCompletionForElement(e: Element): string {
+    let autocompletion = `<${e.tagName}`;
+    let tabindex = 1;
+    if (e.attributes.length) {
+      autocompletion += ` $${tabindex++}`;
+    }
+    autocompletion += `>`;
+    // TODO(timvdlippe): once insertionPoints are implemented in analyzer, uncomment this code
```

Comment:
Yeah, I'd just remove the code. No worries that this is unnecessarily complex, I think we're all very happy with code that improves the usability and pleasantness of the ide, even if it's somewhat (or even very!) complex.